REAPER/Plugin/reaper_ultraschall/UpdateCheck.cpp

```diff
@@ -68,33 +68,23 @@ double QueryCurrentDateTimeAsSeconds()
 
 void UpdateCheck()
 {
-   bool updateCheckRequired = false;
-   std::string lastUpdateCheck;
+   bool updateCheckRequired = true;
```

Comment:
This line causes the update check to run even if the user opted out of update checks. Please remove your test code from pull requests.

Reply:
It isn't because, but I agree the place where updateCheckRequired is defined is confusing.